### ISSUE TITLE
Rule hosting - consider an architecture based on unpkg

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#663]
Current process is that when code is pushed to the prod branch, a rule server is built and pushed to an IBM server: https://able.ibm.com/rules/ . However, it feels like a more 'open' location would be better.

I considered publishing to Git pages, but there are bandwidth restrictions that I think would limit our tools. Another option is to publish the engine to NPM and then fetch it through the JSDelivr CDN. Not sure yet if we should publish all archives every time, or just publish the latest archive. Need to consider impact to the tools.

**Comment by marcjohlic** [comment]
@tombrunet did you move the new rule architecture to JSDelivr?   And if so, do the old rules need to be moved also - or can we just close this knowing that they will be migrated to the new architecture?

**Comment by tombrunet** [comment]
@marcjohlic New architecture is still running on our servers. I discovered that JSDelivr won't really work. We can move to unpkg.com, so I'll update the title. This is more about the tools and how they find the archives than the engine delivery/architecture.

**Comment by jimaek** [comment]
Hey let us know why jsDelivr won't work, maybe we could help somehow?

**Comment by tombrunet** [comment]
@jimaek Works fine as far as I know for fetching JavaScript files, but our packaged help documentation can load directly:
[Help file on unpkg](https://unpkg.com/accessibility-checker-engine@0.0.1-rc.5/help/en-US/Rpt_Aria_OrphanedContent_Native_Host_Sematics.html#%7B%22ruleId%22%3A%22Rpt_Aria_OrphanedContent_Native_Host_Sematics%22%2C%22value%22%3A%5B%22VIOLATION%22%2C%22POTENTIAL%22%5D%2C%22path%22%3A%7B%22dom%22%3A%22%2Fhtml%5B1%5D%2Fbody%5B1%5D%2Fdiv%5B1%5D%2Fdiv%5B1%5D%2Fdiv%5B1%5D%2Fbutton%5B1%5D%22%2C%22aria%22%3A%22%2Fdocument%5B1%5D%2Fbutton%5B1%5D%22%7D%2C%22ruleTime%22%3A0%2C%22reasonId%22%3A%22Fail_1%22%2C%22message%22%3A%22Content%20is%20not%20within%20a%20landmark%20element%22%2C%22messageArgs%22%3A%5B%5D%2C%22apiArgs%22%3A%5B%5D%2C%22bounds%22%3A%7B%22left%22%3A1200%2C%22top%22%3A928%2C%22height%22%3A48%2C%22width%22%3A48%7D%2C%22snippet%22%3A%22%3Cbutton%20aria-label%3D%5C%22Back%20to%20Top%5C%22%20type%3D%5C%22button%5C%22%20class%3D%5C%22BackToTopBtn-module--button--3N38o%5C%22%3E%22%2C%22category%22%3A%22Accessibility%22%2C%22ignored%22%3Afalse%2C%22level%22%3A%22violation%22%7D)

JSDelivr advertises that they only return the files with the text/plain mime, so it won't render:
[Same help file on JSDelivr](https://cdn.jsdelivr.net/npm/accessibility-checker-engine@0.0.1-rc.5/help/en-US/Rpt_Aria_OrphanedContent_Native_Host_Sematics.html#%7B%22ruleId%22%3A%22Rpt_Aria_OrphanedContent_Native_Host_Sematics%22%2C%22value%22%3A%5B%22VIOLATION%22%2C%22POTENTIAL%22%5D%2C%22path%22%3A%7B%22dom%22%3A%22%2Fhtml%5B1%5D%2Fbody%5B1%5D%2Fdiv%5B1%5D%2Fdiv%5B1%5D%2Fdiv%5B1%5D%2Fbutton%5B1%5D%22%2C%22aria%22%3A%22%2Fdocument%5B1%5D%2Fbutton%5B1%5D%22%7D%2C%22ruleTime%22%3A0%2C%22reasonId%22%3A%22Fail_1%22%2C%22message%22%3A%22Content%20is%20not%20within%20a%20landmark%20element%22%2C%22messageArgs%22%3A%5B%5D%2C%22apiArgs%22%3A%5B%5D%2C%22bounds%22%3A%7B%22left%22%3A1200%2C%22top%22%3A928%2C%22height%22%3A48%2C%22width%22%3A48%7D%2C%22snippet%22%3A%22%3Cbutton%20aria-label%3D%5C%22Back%20to%20Top%5C%22%20type%3D%5C%22button%5C%22%20class%3D%5C%22BackToTopBtn-module--button--3N38o%5C%22%3E%22%2C%22category%22%3A%22Accessibility%22%2C%22ignored%22%3Afalse%2C%22level%22%3A%22violation%22%7D)

It's an understandable restriction, but

**Comment by jimaek** [comment]
Ok I understand, if anything changes let me know :)

**Comment by tombrunet** [comment]
This should be closed with the latest extension release, and will soon be enabled as a release candidate for CI tools.

@jimaek FYI - for now, we decided to pull our js/json files from jsdelivr and our help files from unpkg.
